test/MusicStore.Test/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "compilationOptions": {
+    "warningsAsErrors": "true"
+  },
+  "dependencies": {
+    "MusicStore": { "type": "default", "version": "1.0.0-*" },
```

Comment:
You can just remove all the special stuff from here. Just `"MusicStore": "",` I think should work (it's a project-to-project reference).